frameworks/hdfs/tests/config.py

```diff
@@ -103,7 +103,7 @@ def run_hdfs_command(service_name, command):
     """
     Execute the command using the Docker client
     """
-    full_command = 'docker run -e HDFS_SERVICE_NAME={service_name} {image_name} /bin/bash -c "/configure-hdfs.sh && {cmd}"'.format(
+    full_command = 'docker run -e HDFS_SERVICE_NAME={service_name} {image_name} /bin/bash -c "hadoop-*/configure-hdfs.sh && {cmd}"'.format(
```

Comment:
We could have left `configure-hdfs.sh` in the root and then this `*` here would not be required.

Reply:
Yeah, I don't have a strong preference for either way, just thought having everything in a specific, non-root working directory is cleaner.

Reply:
Note that #2327 was created to address this script having moved.